docs/integrations/sources/snapchat-marketing.md

```diff
@@ -0,0 +1,91 @@
+# Snapchat Marketing API
+
+## Overview
+
+The Snapchat Marketing source can sync data from the [Snapchat Marketing API](https://marketingapi.snapchat.com/docs/)
+
+Useful links:
+- [Snapchat Ads Manager](https://ads.snapchat.com/)  
+- [Snapchat API Docs](https://marketingapi.snapchat.com/docs/)
+- [Snapchat API FAQ](https://businesshelp.snapchat.com/s/article/api-faq?language=en_US)
+- [Set up Snapchat Business account](https://businesshelp.snapchat.com/s/article/get-started?language=en_US)
+- [Activate Access to the Snapchat Marketing API](https://businesshelp.snapchat.com/s/article/api-apply?language=en_US)
+
+#### Output schema
+
+This Source is capable of syncing the following Streams:
+
+- [Organization](https://marketingapi.snapchat.com/docs/#organizations)
+- [Ad Account](https://marketingapi.snapchat.com/docs/#get-all-ad-accounts) (Incremental)
+- [Creative](https://marketingapi.snapchat.com/docs/#get-all-creatives) (Incremental)
+- [Media](https://marketingapi.snapchat.com/docs/#get-all-media) (Incremental)
+- [Campaign](https://marketingapi.snapchat.com/docs/#get-all-campaigns) (Incremental)
+- [Ad](https://marketingapi.snapchat.com/docs/#get-all-ads-under-an-ad-account) (Incremental)
+- [Ad Squad](https://marketingapi.snapchat.com/docs/#get-all-ad-squads-under-an-ad-account) (Incremental)
+- [Segments](https://marketingapi.snapchat.com/docs/#get-all-audience-segments) (Incremental)
+
+#### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `integer` | `integer` |  |
+| `number` | `number` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+| `boolean` | `boolean` |  |
+
+#### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental - Append Sync | Yes |
+| Namespaces | No |
+
+### Requirements
+
+* client_id - Snapchat account client ID
+* client_secret - Snapchat account client secret
+* refresh_token - Snapchat account refresh token 
+
+### Setup guide
+
+To get the required credentials you need to set up a snapchat business account.
+Follow this guide to set up one: [Set up Snapchat Business account](https://businesshelp.snapchat.com/s/article/get-started?language=en_US)
+After that - [Activate Access to the Snapchat Marketing API](https://businesshelp.snapchat.com/s/article/api-apply?language=en_US)  
+Adding the OAuth2 app requires the `redirect_url` parameter. If you have the API endpoint that will handle next OAuth process - write it to this parameter. 
+If not - just use some valid url. Here's the discussion about it: [Snapchat Redirect URL - Clarity in documentation please](https://github.com/Snap-Kit/bitmoji-sample/issues/3) 
+On this step you will retrieve **Client ID** and **Client Secret** carefully save **Client Secret** - you cannot view it in UI, only by regenerating
```

Comment:
Use `*` or numbers to bullet point these instructions, formatting in rich view is a bit off at the moment

Reply:
Done.